docs/index.md

```diff
@@ -1 +1,293 @@
+---
+layout: ""
+page_title: "JFrog Xray Provider"
+description: |-
+  The Xray provider is used to interact with the resources supported by JFrog Xray.
+---
+
 # JFrog Xray Provider
+
+The [Xray](https://jfrog.com/xray/) provider is used to interact with the
+resources supported by JFrog Xray. Xray is a part of JFrog Artifactory and can't be used separately.
+The provider needs to be configured with the proper credentials before it can be used.
+Xray API documentation can be found [here](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API)
+
+Links to documentation for specific resources can be found in the table of contents to the left.
+
+This provider requires access to Artifactory and Xray APIs, which are only available in the _licensed_ pro and enterprise editions.
+You can determine which license you have by accessing the following URL
+`${host}/artifactory/api/system/licenses/`
+
+You can either access it via api, or web browser - it does require admin level credentials, but it's one of the few APIs that will work without a license (side node: you can also install your license here with a `POST`)
+
+```bash
+curl -sL ${host}/projects/api/system/licenses/ | jq .
+{
+  "type" : "Enterprise Plus Trial",
+  "validThrough" : "Jan 29, 2022",
+  "licensedTo" : "JFrog Ltd"
+}
+```
+
+The following 3 license types (`jq .type`) do **NOT** support APIs:
+- Community Edition for C/C++
+- JCR Edition
+- OSS
+
+## Example Usage
+
+```terraform
+# Required for Terraform 0.13 and up (https://www.terraform.io/upgrade-guides/0-13.html)
+terraform {
+  required_providers {
+    xray = {
+      source  = "registry.terraform.io/jfrog/xray"
+      version = "0.0.1"
+    }
+  }
+}
+
+provider "xray" {
+  // Supply the following env vars:
+  // ARTIFACTORY_URL (or JFROG_URL / PROJECTS_URL)
+  // XRAY_ACCESS_TOKEN or JFROG_ACCESS_TOKEN
+}
+
+resource "random_id" "randid" {
+  byte_length = 2
+}
+
+resource "xray_security_policy" "security1" {
+  name        = "test-security-policy-severity-${random_id.randid.dec}"
+  description = "Security policy description"
+  type        = "security"
+  rules {
+    name     = "rule-name-severity"
+    priority = 1
+    criteria {
+      min_severity = "High"
+    }
+    actions {
+      webhooks = []
+      mails    = ["test@email.com"]
```

Comment:
is this V1 or V2?

Reply:
V2. Mails is this field in the UI: 
![image](https://user-images.githubusercontent.com/55418063/144903541-b9266c29-b997-4986-aad6-09f60feef910.png)